zerver/actions/message_delete.py

```diff
@@ -39,6 +39,7 @@ def check_update_first_message_id(
         property="first_message_id",
         value=stream.first_message_id,
         stream_id=stream.id,
+        name=stream.name,
```

Comment:
Why are we adding this? I guess do existing `stream` events include the field? I guess we're probably going to remove that in bulk later, so seems OK.

Reply:
Yes, existing events pass this and API docs also mentions this. And `apply_event` also find stream object to update using name and not ID. So, yeah should be better to remove all of them together.